students/KyleBarry/session02/grid_printer.py

```diff
@@ -0,0 +1,39 @@
+def grid():
+    """Part I: print the grid"""
+    plus = '+'
+    hyphen = ' -'
+    empties = '  '
+    
+    plus_line = plus + (hyphen*4) + ' ' + plus + (hyphen*4) + ' ' + plus + '\n'
+    no_plus_line = '|' + empties*4 + ' ' + '|' + empties*4 + ' ' + '|' + '\n'
+    
+    print(plus_line + (no_plus_line*4) + plus_line +
+    (no_plus_line*4) + plus_line)
+
+def grid_adjusted(n):
+    """Part II: adjust the size of the grid"""
+    plus = '+'
```

Comment:
not sure there is much point in using a variable for a single charater. unless you are thinking that you might want to change it to a different character -- if that's the case, then use a name that makes sense if you change it -- like "corner" maybe?